docker/production/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Dockerfile to run the PCIC data portal   #
 ############################################
 
-FROM pcic/pdp-base-minimal:1.0.0
+FROM pcic/pdp-base-minimal:pdp-python3
```

Comment:
Add a comment that this should be replaced by a release tag when that branch has been merged.